.gitignore

```diff
@@ -88,3 +88,5 @@ var/lib/node_*
 .DS_Store
 
 !*.swagger.*
+
+ZScaler*
```

Comment:
Do we know why we added this?

Reply:
purely debug stuff. I was building local docker images and our Zscaler policy requires installing there certificates or all internet connections will be denied

Reply:
removed